config/utils.go

```diff
@@ -36,18 +36,14 @@ func desanitize(actual, target *model.Config) {
 		*target.SqlSettings.DataSource = *actual.SqlSettings.DataSource
 	}
 
-	if len(target.SqlSettings.DataSourceReplicas) == 1 && target.SqlSettings.DataSourceReplicas[0] == model.FAKE_SETTING {
```

Comment:
The changes in this file are reverting the previous approach of sanitization/desanitization.